server/plugin/command.go

```diff
@@ -85,6 +85,46 @@ func (p *Plugin) getGithubClient(userInfo *GitHubUserInfo) *github.Client {
 	return p.githubConnect(*userInfo.Token)
 }
 
+func (p *Plugin) muteGithubUser(_ *plugin.Context, args *model.CommandArgs, parameters []string, userInfo *GitHubUserInfo) string {
+	mutedUsernameBytes, _ := p.API.KVGet(userInfo.UserID + "-muted-users")
```

Comment:
Why is this error not checked?

Reply:
`KVGet` returns an empty byte slice, rather than a nil one, even in the event of an error. Since I'm checking the length below I didn't think it mattered if there was an error. I added a check to be safe.